src/VisualStudioEditorsSetup/ProvideBindingRedirectionAttribute.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using Microsoft.VisualStudio.Shell;
+
+namespace Microsoft.VisualStudio
+{
+    /// <summary>
+    ///     A <see cref="RegistrationAttribute"/> that provides binding redirects for the project system.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Assembly, AllowMultiple = true)]
+    internal sealed class ProvideEditorsBindingRedirectionAttribute : RegistrationAttribute
```

Comment:
Instead of creating another attribute, please move the existing one to the Common directory and link it from both VSIXs.